pkg/node/node.go

```diff
@@ -519,10 +519,15 @@ func (s *CSINodeService) NodeUnpublishVolume(ctx context.Context, req *csi.NodeU
 	var pod corev1.Pod
 	// initialize it to avoid lint error: considering preallocating (prealloc)
 	owners := []string{}
+	var podFoundErr bool
 	for _, owner := range volumeCR.Spec.Owners {
 		if err := s.k8sClient.ReadCR(ctx, owner, volumeCR.Namespace, &pod); err != nil {
+			// CSI SPEC natively does not contain pod information in NodeUnpubishVolume request
+			// We use a pod owner filter to find it. But in case no pod found due to some unexpected
+			// behavior should not block NodeUnpublishVolume, we should cotinue to finish volume unpublish
+			// Here we just log an error
 			ll.Errorf("Unable to read volume owner pod information: %s, %v", owner, err)
-			return nil, status.Error(codes.Internal, err.Error())
```

Comment:
In test-short-ci, in which case it goes into this failure case?

Reply:
please see this log, https://asd-ecs-jenkins.isus.emc.com/job/csi-master-ci/1051/artifact/tests/log.txt.

Reply:
It seems that, with this change, if there still exists another running pod mount with this volume, with the pod to be removed not found and the podFoundErr set to true, the volume would also be turned to VolumeReady status. 

Would it be better to further change the logic here to directly use "continue" (i.e. don't add this pod to the owners list) if the pod cannot be found from k8s instead of using the condition "len(volumeCR.Spec.Owners) == 0 || podFoundErr" to determine whether to change volume status to VolumeReady?

Reply:
@CraneShiEMC , Good suggestions. updated the patch.